docs/implement-a-customized-actuator-en.md

```diff
@@ -0,0 +1,322 @@
+# Custom SumActuator
+
+Having a tailored-made actuator is key to building a java-tron-based customized public chain. This article illustrates how to develop a java-tron-based `SumActuator`.
```

Comment:
DO NOT invent new words: ~~tailored-made~~.
Just put your invention to google and you'll find the result.